### ISSUE TITLE
american government - references

### ISSUE THREAD

**Issue by lwtchu** [openstax/cnx-recipes#3244]
Within American Government 2e, there are references in the chapter that collate to the end of the book. The mechanics should be already functional and working in the recipe. 

# Within text
![Screen Shot 2020-12-21 at 11 22 44 AM](https://user-images.githubusercontent.com/11299125/102804077-df866800-437e-11eb-9aa8-77df35988e2f.png)

## Mark up
```
<para id="fs-id1163755410166">Under the <term>council-administrator system</term>, the voters elect council members to serve for a specified period of time, and the council in turn appoints an administrator to oversee the operation of the government. The administrator serves at the directive of the council and can be terminated by the council. The goal of this arrangement is to divide administrative and policymaking responsibilities between the elected council and the appointed administrator.<cite target-id="rf-77"><note class="reference" display="inline" id="rf-77">Coppa, <emphasis effect="italics">County Government: A Guide to Efficient and Accountable Government</emphasis>.</note></cite></para>
```

## Specs
color: standard link color
text treatment: NO UNDERLINE

# End of book
![Screen Shot 2020-12-21 at 11 37 19 AM](https://user-images.githubusercontent.com/11299125/102805489-e7dfa280-4380-11eb-9c5d-8e497d43e49e.png)

## Specs
- Everything should follow the basic header specs #3164 
- Is it possible to reduce spacing after each note so that it isn't double spaced?
- Have second/third/etc line text start where the first line starts

**Comment by wilkus27** [comment]
@lwtchu That's how it looks:

![image](https://user-images.githubusercontent.com/40228854/105352750-95e3a400-5bee-11eb-8b9d-cfaa1997a147.png)

Is it alright?

**Comment by lwtchu** [comment]
@wilkus27 can we make the numbers in the list unbold? or is that how it's set in the content?

**Comment by wilkus27** [comment]
@lwtchu bold was added in styles. This is how it looks without it:

![image](https://user-images.githubusercontent.com/40228854/105468162-f4fcf380-5c96-11eb-8064-e5ac9ebcf0d9.png)

**Comment by lwtchu** [comment]
@wilkus27 awesome. sorry I just noticed this and i have one more tiny thing - can the numbers be aligned right so the text can be all aligned? 

![Screen Shot 2021-01-22 at 11 42 18 AM](https://user-images.githubusercontent.com/11299125/105525639-fcc49400-5ca6-11eb-9e98-915c5d5b1786.png)

tried my best to draw. let me know if it's not super clear 😓

**Comment by wilkus27** [comment]
@lwtchu This is how it looks:

![image](https://user-images.githubusercontent.com/40228854/105693578-2debd100-5f00-11eb-9077-e171a79418a9.png)

I resolved it in that way: I set the same width for numbers (2.2rem) and align text to right. So now padding between number and text is added to this width no to the width of number. But as I checked, we have never applied such solution. In every book numbers are aligned to the left. Do you think we should fix it in all books? Or just make an exception here?

**Comment by lwtchu** [comment]
@wilkus27 i think we should fix it in all the books - would that be a lot to fix? we can space out the testing.

**Comment by wilkus27** [comment]
@lwtchu this is how it looks in Psychology:

![image](https://user-images.githubusercontent.com/40228854/106143553-3bf55800-6173-11eb-939c-50641b67b944.png)

There are no numbers. Is it intentional?

**Comment by lwtchu** [comment]
@wilkus27 yes i think so. because these don't have a reference in the text and it's just a list of references. this looks fine for psychology

**Comment by wilkus27** [comment]
@lwtchu Based on the informations from https://github.com/openstax/cnx-recipes/issues/3422#issue-792274922  I added indentations to References. This is how it looks:

![image](https://user-images.githubusercontent.com/40228854/107615069-cfd32380-6c4b-11eb-9c55-0cbca23f8542.png)

I applied it to Economics and Psychology.

**Comment by lwtchu** [comment]
@wilkus27 thank you for catching i think this looks good 🤔  i'll update the original comment to include this note
